ydb/core/tx/schemeshard/schemeshard__stats.h

```diff
@@ -13,20 +13,22 @@ namespace NSchemeShard {
 struct TStatsId {
     TPathId PathId;
     TTabletId Datashard;
+    ui32 FollowerId;
 
-    TStatsId(const TPathId& pathId, const TTabletId datashard = TTabletId(0))
+    TStatsId(const TPathId& pathId, const TTabletId datashard = TTabletId(0), ui32 followerId = 0)
         : PathId(pathId)
         , Datashard(datashard)
+        , FollowerId(followerId)
     {
     }
 
     bool operator==(const TStatsId& rhs) const {
-        return PathId == rhs.PathId && Datashard == rhs.Datashard;
+        return PathId == rhs.PathId && Datashard == rhs.Datashard & FollowerId == rhs.FollowerId;
     }
 
     struct THash {
         inline size_t operator()(const TStatsId& obj) const {
-            return MultiHash(obj.PathId.Hash(), obj.Datashard);
+            return MultiHash(MultiHash(obj.PathId.Hash(), obj.Datashard), obj.FollowerId);
```

Comment:
I don't think It's good idea (combine hashes also needed).
But you have more easy way to do it
```suggestion
            return MultiHash(obj.PathId.Hash(), obj.Datashard, obj.FollowerId);
```

Reply:
Fixed